README.md

```diff
@@ -17,19 +17,36 @@ This repository contains the solver and the backend.
 1. Clone the project.
 
 ```bash
-git clone https://github.com/Project-Summer-AI-Lab-Glider/nurse-scheduling-problem-solver.git
+$ git clone https://github.com/Project-Summer-AI-Lab-Glider/nurse-scheduling-problem-solver.git
 ```
 
 2. Enter the project directory:
 
 ```bash
-cd nurse-scheduling-problem-solver
+$ cd nurse-scheduling-problem-solver
 ```
+3. Install dependecies
+```
+$ julia 
+julia> ]
+(@v1.5) pkg> activate .
+(nurse-scheduling-problem-solver) pkg> instantiate
+```
+If you will encouter errors like below:
+```
+ERROR: `PACKAGE` is a direct dependency, but does not appear in the manifest. If you intend `PACKAGE` to be a direct dependency, run `Pkg.resolve()` to populate the manifest. Otherwise, remove `PACKAGE` with `Pkg.rm("PACKAGE")`. Finally, run `Pkg.instantiate()` again.
+```
+while installing dependecies, try run:
+```
+(nurse-scheduling-problem-solver) pkg> resolve
+(nurse-scheduling-problem-solver) pkg> instantiate
+```
```

Comment:
Nie no, nie dodawajmy tego. Z tego co rozumiem to jak dostaniesz taki błąd to wyświetla się instrukcja co zrobić. Wydaje mi się, że to jest zbędne w naszym README

Reply:
To też zależy od tego, kto docelowo będzie używać README.
Bo jeżeli to nie będą osoby które wcześniej nie miały do czynienia z Julią, to ta instrukcja może być trochę nieintuicyjna. Przynajmniej dla mnie była. Ale ja bym zapytał też o opinii innych.
@pectom , @maciekb05 , bo wy też stawialiście sobie serwer. Jak sądzicie, zostawiać to czy nie ?

Reply:
no ale w instrukcji tlumaczysz to co jest napisane w "errorze"

Reply:
Nie do końca, bo w errorze są 2 ścieżki @Qwebeck określa jedną z nich. Ja miałem problem z odpaleniem serwera, ale to głównie przez wersję Julii. Tutaj też miałem zagwozdkę, ale szybko znalazłem rozwiązanie.

Z tym, że jeśli by to było na repo to miałbym mniej szukania, więc jestem za zostawieniem tego. Jest to instrukcja trochę dla dziecka, które nigdy wcześniej nie miało styczności z Julia. Sam takim jestem i raczej tylko takie dzieci będą w naszym projekcie, oszczędźmy im tę chwilkę XD

Reply:
jakie scieżki? zgadzam się, że wersja Julii by sie przydala

Reply:
Otherwise, remove `PACKAGE` with `Pkg.rm("PACKAGE")

Reply:
Ja nie rozumiem, ale czy ten komunikat nie wyskakuje automatycznie? (mi się cos takiego nie zdazylo) Bo jak tak to nie ma co go wklejac do readme

Ten:
```ERROR: `PACKAGE` is a direct dependency, but does not appear in the manifest. If you intend `PACKAGE` to be a direct dependency, run `Pkg.resolve()` to populate the manifest. Otherwise, remove `PACKAGE` with `Pkg.rm("PACKAGE")`. Finally, run `Pkg.instantiate()` again.```

Reply:
Ta, wyskakuje automatycznie. Tylko, że takie ogórki jak my nie wiedzą co z tym zrobić - to po pierwsze. A po drugie to tam masz opisane 2 sposoby na rozwiązanie problemu: resolve -> instantiate albo rm("PACKAGE") -> instantiate. Wiem, że drugie nie ma sensu, ale jednak jest taka podpowiedź i o to mi chodziło z tymi ścieżkami

Reply:
Zmienione na inna wersję